.travis.yml

```diff
@@ -338,7 +367,7 @@ script:
     - make $TRAVIS_PAR_MAKE check TESTS=
     - $SOS_PM_PRE
     - >
-      if [[ $SOS_TRANSPORT_OPTS = *"with-ofi"* ]]; then
+      if [[ $SOS_TRANSPORT_OPTS = *"with-ofi"* ]] || [[ $SOS_TRANSPORT_OPTS = *"with-ucx" ]]; then
```

Comment:
We might need a `*` at the end of `"with-ucx"` as the flag will definitely have a value.

Reply:
Whoops!  Thanks - it probably wasn't running then...